v2/blockstore/readonly.go

```diff
@@ -165,7 +165,7 @@ func (b *ReadOnly) AllKeysChan(ctx context.Context) (<-chan cid.Cid, error) {
 
 	ch := make(chan cid.Cid, 5)
```

Comment:
Where does the number 5 come from?

Reply:
I wondered the same; this originates from @willscott's carbs at https://github.com/willscott/carbs/blob/1036c89e1fd753bc375d860725c9e29ab85b72ac/carbs.go#L161.

My guess is he wanted the channel to have some buffering for the sake of performance. I imagine that, unless measured, we should stick to a buffer size of 1, or no buffering. Any other buffer size should be carefully selected and documented, in my opinion :)

Reply:
Obligatory thread: https://twitter.com/erentantekin/status/1330957563496525824

Reply:
Agreed 👍

Reply:
I've added a TODO about this.